staging/src/k8s.io/apimachinery/pkg/util/sets/doc.go

```diff
@@ -14,7 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by set-gen. DO NOT EDIT.
-
-// Package sets has auto-generated set types.
+// Package sets has generic set and specified sets.
```

Comment:
Please explain that the goal is to switch to generic sets over time and specific ones are deprecated.